### PR TITLE
Pal 1025 s3 k8s config

### PR DESCRIPTION
**Pull request by dev930018**

### CODE REVIEW

**Review by pd104923 — CHANGES_REQUESTED** [review]
A couple of questions/suggestions

File: s3-reader/src/main/java/uk/gov/gchq/palisade/service/data/s3/S3Properties.java [hunk 1]
```diff
@@ -16,67 +16,14 @@
...
-import java.util.Optional;
-
 /**
- * Properties used in communicating with the AWS S3 Bucket.
+ * S3 Properties class, containing default information about resources added to S3.
  */
-@ConfigurationProperties(prefix = "s3")
-public class S3Properties {
-
+public final class S3Properties {
```
[the remark this answers is not in the document]
Comment (dev930018) [reply s3-reader/src/main/java/uk/gov/gchq/palisade/service/data/s3/S3Properties.java:22]: Nope

**Review by pd104923 — CHANGES_REQUESTED** [review]
Looks good, just the license changes that need to be reverted.

**Review by pd104923 — DISMISSED** [review]

File: s3-resource-service/src/main/resources/application-s3.yaml [hunk 2]
```diff
@@ -51,9 +51,9 @@ alpakka.s3:
   # Custom endpoint url, used for alternate s3 implementations
   # To enable virtual-host-style access with Alpakka S3 use the placeholder `{bucket}` in the URL
   # eg. endpoint-url = "http://{bucket}.s3minio.alpakka:9000"
-  endpoint-url: https://{bucket}.amazonaws.com
+  # endpoint-url: https://{bucket}.amazonaws.com
 
-  path-style-access: false
+  # path-style-access: false
```
Comment (developer01189998819991197253) [inline s3-resource-service/src/main/resources/application-s3.yaml:56]: path style access is deprecated, update!

**Review by developer01189998819991197253 — CHANGES_REQUESTED** [review]

**Review by developer01189998819991197253 — APPROVED** [review]

**Review by pd104923 — APPROVED** [review]

**Pull request merged**